index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Dancing+Script&family=Roboto:wght@700&family=Zen+Dots&display=swap" rel="stylesheet">
+</head>
+<body>
+    <main class="wrapper">
+    <header>
+        <h1>My Logo</h1>
+        <img src="Union.png" alt="navigation icon">
+    </header>
```

Comment:
- Great job using a `<header>` tag as this represents the header of your portfolio👍🏽 

- I'd like to suggest that your `<h1>My Logo</h1>` should NOT be an `<h1>` because this is not the main header text of your site. You can use anyother `<hx>` tag or a `<p>` tag preferrably. 
- Also, it isimportant to wrap the `My Logo` in an `<a>` tag because as this is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.
- Your hamburger menu will be better off if it were to be in a `<nav>` bar. The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.

At the end, your code should look something like this 👇🏽 

```
    <header>
        <p><a href="">My Logo</a></p>
       <nav>
         <img src="Union.png" alt="navigation icon">
       </nav>
    </header>
```

- You are free to modify it by adding class names or id's as you wish. 🤗